From bc9f38089840ce68f3c7d5bf2f2793b827d28463 Mon Sep 17 00:00:00 2001
From: mnacakanov <petrushksa@gmail.com>
Date: Sun, 21 Dec 2025 10:30:48 -0500
Subject: [PATCH 04/15] Add BT support for PRIME B650M-A AX6 II motherboard

By default BT starts up but doesn't show any devices when you try to pair. Adding the device id in block list, fixes that.
---
 drivers/bluetooth/btusb.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/bluetooth/btusb.c b/drivers/bluetooth/btusb.c
index dc7191bf4..d2d2b2a9a 100644
--- a/drivers/bluetooth/btusb.c
+++ b/drivers/bluetooth/btusb.c
@@ -587,6 +587,8 @@ static const struct usb_device_id quirks_table[] = {
 						     BTUSB_WIDEBAND_SPEECH },
 	{ USB_DEVICE(0x13d3, 0x3612), .driver_info = BTUSB_REALTEK |
 						     BTUSB_WIDEBAND_SPEECH },
+	{ USB_DEVICE(0x0489, 0xe112), .driver_info = BTUSB_REALTEK |
+						     BTUSB_WIDEBAND_SPEECH },
 	{ USB_DEVICE(0x0489, 0xe122), .driver_info = BTUSB_REALTEK |
 						     BTUSB_WIDEBAND_SPEECH },
 
-- 
2.56.0

